client/Contexts/Introspect/mock/expectations-basic.ts

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+import {Expectations} from "../ExpectationTypes";
```

Comment:
Minor concerns that this duplicates a lot of the real `Expectations` - is it needed or can it be reduced to provide a smaller test case (e.g the reducers in the callbacks)

Reply:
Yeah, I removed the real Expectations - we haven't actually determined the real ones yet. This one purely exists for unit testing this code. We don't want to merge them as that would mean that changes to the real expectations incorrectly break tests.